src/test/java/core/basesyntax/services/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.services.interfaces.FileReader;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String PATH_TO_FILE = "src/test/java/core/basesyntax/sources/"
+            + "test_report.csv";
+    private static final String WRONG_PATH_TO_FILE = "src/test/java/core/basesyntax/sources/"
+            + "fail_test_report.csv";
+    private static final String EXPECTED_TITLE = "type,fruit,quantity";
+    private static final String EXPECTED_SECOND_LINE = "b,banana,20";
+    private static final int EXPECTED_LINES_COUNT = 3;
```

Comment:
these lines don't repeate in the code and are related only to particular test, move them to locar variables expected. Because with the first and second naming it might be confusing if you add more tests, change sequence of code etc

Reply:
done